root/components/list/ReleaseList.js

```diff
@@ -136,21 +159,27 @@ const ReleaseList = ({
         ...(labelsColumn ? [labelsColumn] : []),
         catnosColumn,
         barcodeColumn,
+        ...(showLanguages ? [releaseLanguageColumn] : []),
         ...(instrumentUsageColumn ? [instrumentUsageColumn] : []),
+        ...(typeColumn ? [typeColumn] : []),
+        ...(statusColumn ? [statusColumn] : []),
         ...($c.session?.tport == null ? [] : [taggerColumn]),
         ...(showRatings ? [ratingsColumn] : []),
       ];
     },
     [
-      $c.session,
+      $c.session?.tport,
```

Comment:
This change seems to be unrelated, it is not documented in commit https://github.com/metabrainz/musicbrainz-server/pull/1619/commits/1cfccb902a3b9fba2793a40e70a3fd08ff6d7a16 at least.

Reply:
That's just part of the expected elements in that array as per eslint, I guess I can split it into its own commit if needed.

Reply:
Why wouldn’t it depend on `$c.session?.merger`?

Reply:
Because $c.session?.tport is the only conditional in the table return AFAICT. I don't see $c.session?.merger anywhere in the file actually? :)

Reply:
Ok, it’s from https://github.com/metabrainz/musicbrainz-server/commit/1cfccb902a3b9fba2793a40e70a3fd08ff6d7a16#diff-3070f82643a68f5cadc699007b7fea0d20a5b989184430f6b71af9dc88f95e3cL140-R166 :)
Then why was it set to `$c.session` before?

Reply:
No idea! :D I mean, it should work fine if it changes every time $c.session changes, but I'm not sure why that was set like that.

Reply:
Ok, thanks :) It would be better fixed in a separate commit if possible.